File: shared/src/main/java/com/batu/shared/dto/response/AccountViewDto.java
package com.batu.shared.dto.response;

import java.math.BigDecimal;
import java.time.Instant;
import java.util.UUID;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class AccountViewDto {

    private UUID accountId;
    private String institutionName;
    private String accountName;
    private BigDecimal currentBalance;
    private String accountType;
    private String accountMask;
    private Instant createdAt;
}

File: account-service/src/main/kotlin/com/batu/account_service/dto/AccountViewDto.kt
package com.batu.account_service.dto

import java.util.UUID

import java.math.BigDecimal
import java.time.LocalDateTime

data class AccountViewDto(
    val accountId : UUID,
    val connectionId : UUID,
    val accountName : String,
    val currentBalance : BigDecimal,
    val accountType : String,
    val accountMask : String,
    val createdAt: LocalDateTime?
)